src/RpcClient/TransactionManager.cs

```diff
@@ -104,7 +104,14 @@ private long EstimateNetworkFee()
             foreach (var hash in hashes)
             {
                 size += 166;
-                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES64] + ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES33] + InteropService.GetPrice(InteropService.Neo_Crypto_ECDsaVerify, null);
+                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHNULL]; // message
+                using (ScriptBuilder sb = new ScriptBuilder())
+                {
+                    networkFee += ApplicationEngine.OpCodePrices[(OpCode)sb.EmitPush(64).ToArray()[0]]; // signature
+                    networkFee += ApplicationEngine.OpCodePrices[(OpCode)sb.EmitPush(33).ToArray()[0]]; // pubKey
+                    size += sb.ToArray().Length;
```

Comment:
Before we were not considering this in the size, right?

Reply:
I am not sure if it's included in this 166

Reply:
```
if (witness_script.IsSignatureContract())
 {
    size += 67 + witness_script.GetVarSize();
    networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHDATA1] + ApplicationEngine.OpCodePrices[OpCode.PUSHDATA1] + ApplicationEngine.OpCodePrices[OpCode.PUSHNULL] + InteropService.GetPrice(InteropService.Crypto.ECDsaVerify, null);
 }
```
It's the varsize of invocation and verification script.

Reply:
Could you proceed with the change @chenzhitong?